datachannel.go

```diff
@@ -44,6 +44,8 @@ type DataChannel struct {
 	// OnError             func()
 
 	onMessageHandler    func(DataChannelMessage)
+	onceMutex           sync.Mutex
```

Comment:
You don't need this mutex. `sync.Once` is already goroutine safe.

Reply:
yes, but check the PR description, is not about the execution but the fact that it can be replaced